components/ButtonSecondary.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <button class="button-secondary" @click="handleClick"><slot></slot></button>
+  <button type="button" class="button-secondary" @click="handleClick">
```

Comment:
what was the need for `type="button"`?
should we also add it to `ButtonPrimary`?

Reply:
I added it because i thought i needed it for more narrow css selection, but it's not needed so i'll remove it here and in the main